metrics_utility/automation_controller_billing/extract/extractor_common.py

```diff
@@ -3,7 +3,78 @@
 import tarfile
 
 
-def write_member(member_path, file_obj, max_size, total_extracted_size):
+class Base:
```

Comment:
we should ideally call this based on the file name, we can rename the file to base, to follow the pattern of the other factories?